### PR TITLE
Content list - custom block

### PR DESCRIPTION
**Pull request by bigslott**

### CODE REVIEW

**Comment by reload-branching-check** [comment]
**Reload Commit Checker**
The following commits does not correctly reference a Jira issue:

|SHA|Message|Author|Date|
|---|-------|------|----|
|54bc60322a2163b42f77f98fc2dd1f0febce771c|Add view mode for paragraph teaser lists|Thomas Slott (bigslott@gmail.com)|2016-10-25T05:15:56Z|

**Comment by reload-branching-check** [comment]
**Reload Commit Checker**
The following commits does not correctly reference a Jira issue:

|SHA|Message|Author|Date|
|---|-------|------|----|
|54bc60322a2163b42f77f98fc2dd1f0febce771c|Add view mode for paragraph teaser lists|Thomas Slott (bigslott@gmail.com)|2016-10-25T05:15:56Z|

File: web/themes/custom/mungo/assets_src/sass/components/badge.scss [hunk 1]
```diff
@@ -6,7 +6,9 @@
     ------------------------------------------------------- */
 .node--type-badge.node--view-mode-full {
 
-  padding: 50px 0;
+  &.node--view-mode-full {
```
Comment (troelslenda) [inline web/themes/custom/mungo/assets_src/sass/components/badge.scss]: det er vel det samme som på linie 7?

File: web/themes/custom/mungo/assets_src/sass/layout/article.scss [hunk 2]
```diff
@@ -63,3 +66,83 @@
...
+    border-left: 5px solid $bluish;
+  }
+
+  // Fields.
+  &--heading {
+    @include list-heading-small;
+
+    // Override of the font-size. Accoding to the
+    // 11px on mobile and 14ox on tablet and desktop.
+    font-size: 11px;
```
Comment (troelslenda) [inline web/themes/custom/mungo/assets_src/sass/layout/article.scss]: men så skal det vel ikke være en list-heading-small? altså hvis der er overrides :)

File: web/themes/custom/mungo/assets_src/sass/mixins/typography.mixins.scss [hunk 3]
```diff
@@ -29,13 +29,13 @@
 
 /* Liste Rubrik - small */
 @mixin list-heading-small {
-  font-family: $std-font;
+  font-family: 14px;
```
Comment (troelslenda) [inline web/themes/custom/mungo/assets_src/sass/mixins/typography.mixins.scss]: skal font-size sættes?

**Pull request merged**